pingpong-go/src/pingpong/pong/Pong.go

```diff
@@ -1,29 +1,35 @@
-package pong 
+package pong
 
 import (
+	"bytes"
 	"fmt"
 	"net/http"
-	"github.com/gorilla/mux"
 	"strconv"
-	"bytes"
+
+	"github.com/gorilla/mux"
 )
 
 func pongHandler(w http.ResponseWriter, r *http.Request) {
 	params := mux.Vars(r)
-	i, _ := strconv.Atoi(params["length"])
+	i, err := strconv.Atoi(params["length"])
+	if err != nil {
+		panic(err)
+	}
 	i -= 4
-	var result bytes.Buffer
-	result.WriteString("po")
-	for ; i > 0; i-- { result.WriteString("o") }
+	result := bytes.NewBufferString("po")
```

Comment:
I wrote following benchmark:
file: bench.go:
package main

import (
    "bytes"
)

func A() bytes.Buffer {
    var buf bytes.Buffer
    buf.WriteString("abc")
    return buf
}

func B() bytes.Buffer {
    buf := bytes.NewBufferString("abc")
    return *buf
}

file: bench_test.go:
package main

import (
    "testing"
)

func BenchmarkA(b *testing.B) {
    for i := 0; i < b.N; i++ {
        _ = A()
    }
}

func BenchmarkB(b *testing.B) {
    for i := 0; i < b.N; i++ {
        _ = B()
    }
}

With following results:
BenchmarkA-4   	10000000	       121 ns/op
BenchmarkB-4   	50000000	        34.5 ns/op
PASS
ok  		3.266s

Showing that my change is even faster.